Where_Is_My_Wife/Assets/_Main/Scripts/Stage/LavaProyectilLeft.cs

```diff
@@ -0,0 +1,15 @@
+using UnityEngine;
+
+public class LavaProyectilLeft : MonoBehaviour
```

Comment:
Remove this script and reuse the other, you can make things go to left using negative velocity.

Reply:
already added it, it's called lavaTrigger